server/app.py

```diff
@@ -1,11 +1,36 @@
 from flask import Flask
+from marshmallow import Schema
+from flask_restful import Api
+
+from api.user_handler import UserResource
+
 from api.ping_handler import ping_handler
 from api.home_handler import home_handler
 
+from models import db, User, UserSchema
+
+
+def create_app():
+    app = Flask(__name__)
+
+    app.register_blueprint(home_handler)
+    app.register_blueprint(ping_handler)
+
+    # Database variables
+    user = 'postgres'
+    pw = '123456'
+    url = 'localhost:5432'
+    db_name = 'team-bluejay'
```

Comment:
these database variables should be coming from a config file, and that config file should be referencing environment variables